Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,219 @@
+{
+	"translatorID": "de0eef58-cb39-4410-ada0-6b39f43383f9",
+	"label": "Gemeinsamer Bibliotheksverbund ISBN",
+	"creator": "Philipp Zumstein",
+	"target": "",
+	"minVersion": "4.0",
+	"maxVersion": "",
+	"priority": 198,
+	"inRepository": true,
+	"translatorType": 9,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-02-08 11:09:49"
+}
+
+/*
+***** BEGIN LICENSE BLOCK *****
+
+Copyright © 2015 Philipp Zumstein
+
+This file is part of Zotero.
+
+Zotero is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Zotero is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+***** END LICENSE BLOCK *****
+*/
+
+/****************************
+import function are here for easy testing:
+*****************************/
+function detectImport() {
+	var item = JSON.parse(Zotero.read());
+	return detectSearch(item);
+}
+
+function doImport() {
+	var item = JSON.parse(Zotero.read());
+	return doSearch(item);
+}
+/*****************************/
+
+function detectSearch(item) {
+	return !!item.ISBN;
+}
+
+function doSearch(item) {
+	//search the ISBN over the SRU of the GBV, and take the result it as MARCXML
+	//documentation: https://www.gbv.de/wikis/cls/SRU
+	var url = "http://sru.gbv.de/gvk?version=1.1&operation=searchRetrieve&query=pica.isb%3D9780521779241&maximumRecords=1%20%3E%3E%20srsru.gbv.de/gvk?version=1.1&operation=searchRetrieve&query=pica.isb=" + ZU.cleanISBN(item.ISBN) + "&maximumRecords=1";
```

Comment:
Huch, yes indeed. I will correct, but it does not to change anything in the test output.